internal/logout/logout.go

```diff
@@ -0,0 +1,33 @@
+package logout
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/afero"
+	"github.com/supabase/cli/internal/utils"
+)
+
+type RunParams struct {
+	Fsys afero.Fs
+}
+
+var loggedOutMsg = "You are now logged out."
+
+func Run(ctx context.Context, stdout *os.File, params RunParams) error {
+	err := utils.RunProgram(ctx, func(p utils.Program, ctx context.Context) error {
```

Comment:
This operation should be instantaneous, I don't think we need to wrap it inside a separate program.

Reply:
Yes. Thanks. I remove the "RunProgram" wrap.